pkg/platform/runtime/buildscript/buildscript.go

```diff
@@ -160,15 +233,38 @@ func valueString(v *buildexpression.Value) string {
 }
 
 func apString(f *buildexpression.Ap) string {
+	if f.Name == reqFuncName {
+		return apReqString(f)
+	}
+
 	buf := bytes.Buffer{}
 	buf.WriteString(fmt.Sprintf("%s(\n", f.Name))
```

Comment:
By keeping spaces, we only need to check for `f.Name == reqFuncName` and the omit the newlines in this method.